python/tidy/servo_tidy/tidy.py

```diff
@@ -991,25 +1001,27 @@ def check_commits(path='.'):
     raise StopIteration
 
 
-def scan(only_changed_files=False, progress=True):
+def scan(only_changed_files=False, progress=True, stylo=False):
```

Comment:
They still need two different arguments, you want `./mach test-tidy --all` to work.